components/integrations/abstract_incoming_hook.test.tsx

```diff
@@ -4,25 +4,38 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
-import AbstractIncomingWebhook from 'components/integrations/abstract_incoming_webhook.jsx';
+import AbstractIncomingWebhook from 'components/integrations/abstract_incoming_webhook';
+import {Team} from 'packages/mattermost-redux/src/types/teams';
+import {IncomingWebhook} from 'mattermost-redux/types/integrations';
+import {TestHelper} from '../../utils/test_helper';
```

Comment:
Same suggestion to reorder imports. I suppose this applies to many test files in this PR

```suggestion
import {Team} from 'mattermost-redux/src/types/teams';
import {IncomingWebhook} from 'mattermost-redux/types/integrations';
import {TestHelper} from 'utils/test_helper';

import AbstractIncomingWebhook from './abstract_incoming_webhook';
```